docs/add-ins/extract-entity-strings-from-an-item.md

```diff
@@ -201,7 +201,7 @@ The remaining sections describe how this sample (default_entities.js file) extra
 
 ## Extracting entities upon initialization
 
-Upon the [Office.initialize](https://docs.microsoft.com/javascript/api/office#initialize-reason-) event, the entities add-in calls the [getEntities](https://docs.microsoft.com/office/dev/add-ins/reference/objectmodel/requirement-set-1.5/Office.context.mailbox.item#getentities--entitiesjavascriptapioutlook15officeentities) method of the current item. The **getEntities** method returns the global variable `_MyEntities` an array of instances of supported entities. The following is the related JavaScript code.
+Upon the [Office.initialize](https://docs.microsoft.com/javascript/api/office#office-initialize) event, the entities add-in calls the [getEntities](https://docs.microsoft.com/office/dev/add-ins/reference/objectmodel/requirement-set-1.5/Office.context.mailbox.item#getentities--entitiesjavascriptapioutlook15officeentities) method of the current item. The **getEntities** method returns the global variable `_MyEntities` an array of instances of supported entities. The following is the related JavaScript code.
```

Comment:
Could remove DMC domain from URL

Reply:
Done (updated all links throughout this article to remove the `https://docs.microsoft.com` from the beginning of the URL).